Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:jessie-slim
+FROM jnewland/git-and-stuff
```

Comment:
I appreciate this change, but I definitely don't want to use a (non library) maintained image. This would be a huge security risk, and also it's hugely likely that it wouldn't be updated. If you have a suggestion for another base image maintained in Docker Library I'd be happy to review further.

Reply:
The good news is that (if you trust this image) you can host and run it yourself. but honestly, it hasn't been updated in 2 years, and that alone is a warning sign - we know for a fact there are CVEs within! On the other hand, a library image (like jessie slim) was last updated 10-14-2019, so there is more assurance there.

Reply:
Good point. Thanks for the feedback.